bugbug/repository.py

```diff
@@ -92,13 +93,13 @@ def hg_log(repo_dir):
     for rev in hglib.util.grouper(template.count('\\0'), out):
         posixtime = float(rev[3].split(b'.', 1)[0])
         dt = datetime.fromtimestamp(posixtime)
-
```

Comment:
Don't remove this newline (better not to change things that are unrelated to the goal of the patch).